cypress/utils/utils.ts

```diff
@@ -1317,7 +1317,7 @@ export function deleteAllArchetypes() {
 
 export function deleteApplicationTableRows(): void {
     navigate_to_application_inventory();
-    cy.wait(1 * SEC);
+    cy.wait(10 * SEC);
```

Comment:
why do we need to wait 10 seconds?

Changing this will have an important impact as a lot of tests are using this method. Also lots of tests use this method and they are not failing so I guess the waiting here is not needed.

Reply:
Could you trigger a jenkins run? I see it is failing in this ci

Thanks!